ssl-config/src/main/java/com/palantir/remoting/ssl/KeyStoreConfiguration.java

```diff
@@ -33,7 +32,7 @@
 public abstract class KeyStoreConfiguration {
 
     @Value.Parameter
-    public abstract Path path();
+    public abstract URI path();
```

Comment:
TrustStoreConfiguration/KeyStoreConfiguration are often used as part of Dropwizard Configuration objects, which in turn are read from disk using a YAML/JSON mapper.